src/privateStore.js

```diff
@@ -65,30 +72,58 @@ class PrivateStore {
    * @param     {String}    hash                        The hash of the private store OrbitDB
    */
   async _sync (hash) {
-    // if hash === null; create a new orbitdb instance.
-    const orbitdb = new OrbitDB(this.ipfs)
-    //TODO - sync the OrbitDB key-value store
+    if (!this.db) {
+      const orbitdb = new OrbitDB(this.ipfs)
+      // the db needs a unique name, we use the hash of the DID + a store specific name
+      const storeName = multihash(this.muportDID.getDid()) + ".datastore"
```

Comment:
maybe give `multihash` a name that clarifies which hash function we are using?

Reply:
Will do.